services/web/project/serializer.py

```diff
@@ -0,0 +1,12 @@
+from flask_marshmallow import Marshmallow
+from .model import Token
+
+ma = Marshmallow()
+
+def configure(app):
+    ma.init_app(app)
+
+class TokenSchema(ma.ModelSchema):
+    class Meta:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in public nested class](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5161591)